src/player.py

```diff
@@ -53,9 +58,23 @@ def update(self, delta_time):
                 * delta_time
             )
 
-        self.primary_weapon.update(delta_time)
-        self.secondary_weapon.update(delta_time)
+        self.primary_weapon.update(delta_time,self.projectile_list, self.beam_list)
```

Comment:
Formatting issues here.

Reply:
Fixed. Wonder why the formatter didn't get this.